data/json/flags.json

```diff
@@ -210,6 +210,12 @@
     "description": "Prevents repairing of this item even if otherwise suitable tools exist.",
     "inherit": false
   },
+  {
+    "id": "MELTS",
+    "type": "json_flag",
+    "context": [ "COMESTIBLE" ],
+    "info": "This food <neutral>melts when not cold</neutral>, and tastes much <good>better</color> when <color_light_cyan>frozen</color>."
```

Comment:
"cold" itself is a item tag, very confusing, you need to reword it a bit.

Reply:
Also, the `</color>` closing tag should be replaced with `</good>` tag.